### PR TITLE
Add a tree to the city

### PR DESCRIPTION
**Pull request by DaoDaoNoCode**

### CODE REVIEW

**Comment by DaoDaoNoCode** [comment]
Juntao Wang
Yanshan Song

File: 01-python/infrastructure/tree.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
-def draw_tree():
-    print("tree not found")
-    return
+def draw_tree(height=5):
+    for i in range(height):
+        print(' ' * (height - i) + '*' * (2 * i + 1))
+    print(' ' * height + "|")
```
Comment (gallettilance) [inline 01-python/infrastructure/tree.py]: when height is 0 you are printing this. Instead you should print nothing is the height is 0

File: 01-python/infrastructure/tree.py [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
-    print("tree not found")
-    return
+def draw_tree(height=5):
+    if height<=0:
+        print("tree not found")
+    else:
+        for i in range(height):
+            print(' ' * (height - i) + '*' * (2 * i + 1))
+        print(' ' * height + "|")
+    return
```
Comment (gallettilance) [inline 01-python/infrastructure/tree.py]: Nit: please add a new line at the end of your file

**Comment by DaoDaoNoCode** [comment]
Have I squashed them? I am not sure whether I squashed them or not. How could I check that?

**Comment by gallettilance** [comment]
@DaoDaoNoCode It looks like you still have 3 commits here. Try following instructions here: https://github.com/wprig/wprig/wiki/How-to-squash-commits :)

**Comment by DaoDaoNoCode** [comment]
> @DaoDaoNoCode It looks like you still have 3 commits here. Try following instructions here: https://github.com/wprig/wprig/wiki/How-to-squash-commits :)

It works, thank you!

**Pull request closed without merging**